tests/publiccloud/transfer_repos.pm

```diff
@@ -53,7 +55,8 @@ sub run {
             assert_script_run("echo $repo | tee -a /tmp/transfer_repos.txt");
         }
         # VM repos.dir support preparation
-        $args->{my_instance}->ssh_assert_script_run("sudo mkdir $repodir;sudo chmod a+w $repodir");
+        $args->{my_instance}->ssh_assert_script_run("sudo mkdir $repodir;sudo chmod 777 $repodir");
+        $args->{my_instance}->ssh_assert_script_run("sudo ls -la /tmp");
```

Comment:
If still needed this, I suggest to use `ls $repodir/..`

Reply:
but anyway I will remove that it was for testing purposes